File: android/employer/app/src/main/java/com/workforcex/employer/api/Notification.java
package com.workforcex.employer.api;

import com.google.gson.annotations.SerializedName;

public class Notification {
    @SerializedName("id")
    public String id;

    @SerializedName("message")
    public String message;

    @SerializedName("read")
    public boolean isRead;

    @SerializedName("createdAt")
    public String createdAt;
}

File: android/worker/shared/src/commonMain/kotlin/com/workforcex/shared/models/Notification.kt
package com.workforcex.shared.models

import com.google.gson.annotations.SerializedName

data class Notification(
    @SerializedName("id")
    val id: String,

    @SerializedName("message")
    val message: String,

    @SerializedName("read")
    val isRead: Boolean,

    @SerializedName("createdAt")
    val createdAt: String,

    @SerializedName("linkType")
    val linkType: String,

    @SerializedName("linkId")
    val linkId: String
)